packages/bsky/src/hydration/label.ts

```diff
@@ -16,10 +17,36 @@ export type { Label } from '../lexicon/types/com/atproto/label/defs'
 
 export type SubjectLabels = {
   isTakendown: boolean
-  labels: Label[]
+  labels: HydrationMap<Label> // src + val -> label
```

Comment:
uri (subject) + val right?

Reply:
All the labels in `labels` should be for the same subject.  We want to ensure that if we encounter the same label twice, we don't include it both times on the subject.  We want to allow the same label value from two different labelers, though.

In short, we're already narrowed by `uri`, and in this step we narrow on `src + val` to make sure labels in the responses are unique by `(uri, src, val)`.